packages/shylu/shylu_node/tacho/src/impl/Tacho_NumericTools_LevelSet.hpp

```diff
@@ -1939,7 +1939,7 @@ class NumericToolsLevelSet : public NumericToolsBase<ValueType, DeviceType> {
                                  s0.rowptrU, s0.colindU, s0.nzvalsU,
                                  rocsparse_indextype_i32, rocsparse_indextype_i32, rocsparse_index_base_zero, rocsparse_compute_type);
       // workspace
-      #if ROCM_VERSION >= 50400
+      #if (ROCM_VERSION >= 50400 && ROCM_VERSION < 60000)
```

Comment:
Looks like this could be made more consistent with similar changes by swapping the order of the check and function api, e.g. change L1942-1945 to be:
```
      #if (ROCM_VERSION >= 60000)
      rocsparse_spmv
      #else
      rocsparse_spmv_ex
```